addons/mrp/report/mrp_report_mo_overview.xml

```diff
@@ -75,7 +75,7 @@
                 <t t-if="line.get('quantity_reserved', False) is not False" t-out="line['quantity_reserved']" t-options="{'widget': 'float', 'precision': line['uom_precision']}"/>
             </td>
             <td t-if="data['show_receipts']" t-attf-class="text-end {{ 'text-' + line['receipt']['decorator'] if line.get('receipt', {}).get('decorator') else '' }}">
-                <span t-if="line.get('receipt')" t-attf-class="{{ 'text-' + line['receipt']['decorator'] }}" t-out="line['receipt']['display']"/>
+                <span t-if="line.get('receipt')" t-attf-class="{{ 'text-' + line['receipt']['decorator'] if line.get('receipt', {}).get('decorator') else '' }}" t-out="line['receipt']['display']"/>
```

Comment:
```suggestion
                <span t-if="line.get('receipt')" t-attf-class="{{ 'text-' + line['receipt']['decorator'] if line['receipt'].get('decorator') else '' }}" t-out="line['receipt']['display']"/>
```
the key `receipt` is already tested in the `t-if`. The line is a little lighter like this